helpers/portal.go

```diff
@@ -34,6 +47,40 @@ type InitCatalogReq struct {
 	OrgId string `json:"org_id"`
 }
 
+type CnameRequest struct {
+	Cname string `json:"cname"`
+}
+
+func SetPortalCname(client http.Client) error {
+	fmt.Println("Setting portal cname")
+
+	cnameReq := CnameRequest{Cname: data.AppConfig.Cname}
+	reqBody, err := json.Marshal(cnameReq)
+	if err != nil {
+		return err
+	}
+	reqData := bytes.NewReader(reqBody)
+
+	req, err := http.NewRequest("PUT", data.AppConfig.DashboardUrl+ApiPortalCnameEndpoint, reqData)
```

Comment:
not crucial but maybe we can https://pkg.go.dev/net/url#JoinPath to create a URL string.